core/controller/src/main/resources/application.conf

```diff
@@ -2,10 +2,38 @@
 include "logging"
 
 # see http://spray.io/documentation/1.2.2/spray-can/configuration/
```

Comment:
Please update the dead link to http://spray.io/documentation/spray-can/configuration/